build/create_release_notes.py

```diff
@@ -165,8 +197,9 @@ def commit_release_notes(filename, new_versions):
 def get_minor_version(version):
     return '.'.join(version.split('.')[:2])
 
-version_header = re.compile('^#+ (\d+(?:\.\d+)+)$') # match only major or minor versions
+version_header = re.compile(r'^#+ (\d+(?:\.\d+)+)$') # match only major or minor versions
```

Comment:
I'm not sure about this comment. Doesn't this match all versions? I think it would need to be:

```py
r'^## (\d+\.\d+)$'
```

To match only major and minor versions. As it is, I think the `version_header` regex is used to match all versions, so I think it's just the comment that's off.

```suggestion
version_header = re.compile(r'^#+ (\d+(?:\.\d+)+)$') # match all version headers, including major or minor headers
```